vars/tdr.groovy

```diff
@@ -13,6 +13,16 @@ def runEndToEndTests(int delaySeconds, String stage, String buildUrl) {
     wait: false)
 }
 
+def runGitSecrets(repo) {
+  sh "set +e"
+  def exitCode = sh(script: "git-secrets --scan", returnStatus: true)
```

Comment:
Just to check - does this print the output to the console so we can track down the secret?

Reply:
Yeah, it does, you get something like this
```
test.txt:1:XXXXXXXXXXXXXXXXXXX
[ERROR] Matched one or more prohibited patterns

```